pkg/kapp/diff/prepare_res_with_periodic_ann.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package diff
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	ctlres "github.com/k14s/kapp/pkg/kapp/resources"
+)
+
+const (
+	MaxDurationAnnKey   = "kapp.k14s.io/max-duration"
```

Comment:
This would become a private variable if we stop using this in the apply stage.

Reply:
Same reason as above. This ann is needed in apply stage to update update-strategy for resources having MaxDurationAnnKey set.